R/list_datasets.R

```diff
@@ -1,10 +1,11 @@
 #' Get a vector of timeseries variable names
 #' 
-#' get_ts_variables() returns a list of all possible timeseries variables, while
-#' get_ts_variables(site) returns the data available to a specific site.
+#' list_datasets() returns a list of all possible timeseries variables, while
```

Comment:
ahh - I see that this is any data that is a child to a site (watershed too). maybe change doc later to clarify.